pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -96,6 +96,7 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.StringVarP(&opts.ImageRegistry, "private-image-registry", "", "", "Private image registry where pull images from. If set, all required images will be downloaded from it, it would be useful in offline installation scenarios.  In addition, you still can use --kube-image-registry to specify the registry for Kubernetes's images.")
+	flags.StringVar(&opts.PullSecrets, "pull-secrets", "", "Pull Secrets are used to pull images from the private registry, could be secret list separated by comma ('--pull-secrets PullSecret1,PullSecret2')")
```

Comment:
```suggestion
	flags.StringSliceVar(&opts.PullSecrets, "pull-secrets", "", "Pull Secrets are used to pull images from the private registry, could be secret list separated by comma ('--pull-secrets PullSecret1,PullSecret2')")
```

Reply:
Should we use a more clear flag, like `--image-pull-secrets`?